server/datastore/mysql/migrations/tables/20240613172616_HostIssuesTable.go

```diff
@@ -25,6 +25,19 @@ func Up_20240613172616(tx *sql.Tx) error {
 	if err != nil {
 		return fmt.Errorf("failed to create host_issues table: %w", err)
 	}
+
+	// Now, populate the table with failing_policies_counts
```

Comment:
Is this here so that the user doesn't have to wait for cron jobs to run after upgrading?

Reply:
Currently, failing policies are calculated in real time. Without this, all the fails are zeroed out after migration until the policies run (or rerun), which looks like a bug. There is no cron for failing policies -- the values in host_issues table should always be correct.